.github/workflows/create-release-pr.yml

```
@@ -0,0 +1,19 @@
+      - master
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  tests:
+    name: Test package
+    uses: ./.github/workflows/_test.yml
+
+  create-release-branch:
```

actionlint

"steps" section is missing in job "create-release-branch"

(syntax-check)